modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/GridDhtPartitionsStateValidator.java

```diff
@@ -49,6 +54,13 @@
     /** Cache shared context. */
     private final GridCacheSharedContext<?, ?> cctx;
 
+    /**
+     * Collection of partitions that did not pass validation.
+     * This collection is supported and updated by coordinator node only.
+     * Represents the following mapping: group id -> set of partitions.
+     */
+    private Map<Integer, Set<Integer>> invalidParts = new ConcurrentHashMap<>();
```

Comment:
It looks like, this field is not used anywhere and can be removed.

Reply:
it used in line 130:
org.apache.ignite.internal.processors.cache.distributed.dht.topology.GridDhtPartitionsStateValidator#validatePartitionCountersAndSizes:130

Reply:
We only write to this collection. I could not find a place where this collection was read. So, it does not make sense to me, to use this map at all.